protos/volumes/v1/volumes.pb.go

```diff
@@ -0,0 +1,855 @@
+//
+//  Copyright 2020 Docker, Inc.
```

Comment:
This can't be Docker. Inc. with just a DCO:

```suggestion
//  Copyright 2020 Docker Compose CLI authors
```